io_scene_godot/converters/__init__.py

```diff
@@ -21,13 +21,17 @@
 from .armature import export_armature_node, export_bone_attachment
 from .animation import export_animation_data
 
-
+# TODO: What about Empties which refer to group instances? (in 2.8 Collection Instances)
```

Comment:
You are right. There is also an issue requesting this addon to support `group instances`, but I haven't got chance to think about it 😅. Just leave a statement like `TODO: need  support for group instances` statement here, it can be dealt with later.

Reply:
There are a few ways we could do collection instances...
- convert the entire collection into meshes, join them into a single mesh (lossy)
- duplicate the collection's hierarchy under an empty spatial node (messy)
- export the collection as a separate escn, refer to it in the main escn as a PackedScene (many-files-y)

Regarding the first of those options - that's what I'd personally want to use for most of my group instances when exporting to Godot, but I can imagine other scenarios where I'd want to use the other 2 methods... So maybe we could allow for all 3 of these scenarios using object.name tags?

However I should note that providing several options for functionality that will rarely be used does seem a bit overkill.